packages/aragon-cli/test/ipfs/daemonAndConfig.test.js

```diff
@@ -1,4 +1,6 @@
 import { serial as test } from 'ava'
+// import { remove, ensureDir } from 'fs-extra'
```

Comment:
Remove commented out code?

Reply:
I think @0x6431346e wanted to be sure to add those on a future PR. Is this still relevant?